spec/requests/collections_spec.rb

```diff
@@ -402,6 +402,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       FactoryBot.create(:miq_search)
       test_collection_query(:search_filters, api_search_filters_url, MiqSearch)
     end
+
+    it 'query shortcuts' do
+      FactoryBot.create(:shortcut)
+      test_collection_query(:shortcuts, api_shortcuts_url, MiqShortcut)
+    end
```

Comment:
Not for this PR, I feel like we should move this into dedicated files.  Yeah, the files will be small, but then the pattern is clearer to people implementing a new endpoint - they see a spec file and they copy the pattern (as opposed to completely missing it in this generic file).